telegram/_dice.py

```diff
@@ -61,7 +61,8 @@ class Dice(TelegramObject):
         emoji (:obj:`str`): Emoji on which the dice throw animation is based.
 
     Attributes:
-        value (:obj:`int`): Value of the dice.
+        value (:obj:`int`): Value of the dice. 1-6 for dice, darts and bowling balls, 1-5 for
+            basketball and football/soccer ball, 1-64 for slot machine.
```

Comment:
We started moving all of those numeric values into the constant file to compare to easier in you code. I think we should do this here as well.

Reply:
I think no, as those values are range of numbers and can vary so we can't include them in constant file

Reply:
it's a good question. Should the constants for this be a `range()` or just `MIN_*_VALUE` and `MAX_*_VALUE`? I think it depends on the use case, @Poolitzer what use case for "comparsion" do you have in mind?

Reply:
Oh yes, use case is an interesting question. I can't really think of one.

Reply:
hmm, the only use case I think of is like: `if update.message.dice.value == constants.DiceLimits.DICE_MAX_VALUE: update.message.reply_text("congrats! You got the highest value!)`

of course the above can be done with subscription (`[-1]`) if it's a range, but having MAX, MIN is more better imo

Reply:
We have a few constants where we just have the max as constant b/c the min value is of no particular interest and/or rather obvious. Would be fine with me here as well, I guess. Otherwise, I'd probably also go for MIN_/MAX_* - Keep It Simple, Stupid 🙆 

In any case, that's out of scope for this PR …